CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ CHANGELOG
 
 **CHANGES**
 - Pyxis is now disabled by default and it must be manually enabled on the head node.
+- Upgrade Python runtime to version 3.12 in ParallelCluster Lambda Layer.
+- Remove version pinning for setuptools to version prior to 70.0.0.
```

Comment:
I think it's worth it to document these dependencies because:
1. Changing the PC Lambda Layer runtime is equivalent to making pcluster compatible with a newer version of Python. When old Lambda runtimes are used, users may receive warnings; this upgrade may solve these warnings and it's useful to keep track of this improvement.
1. setuptools: this package affects the pcluster installation, so if the user faces issues in installing pcluster it may also be because of changes in the new version of setuptools.